.travis/install_tango.sh

```diff
@@ -2,5 +2,10 @@
 
 set -e
 
+if [[ "$CHECK_ABI_API_COMPLIANCE" == "ON" ]]
```

Comment:
Consider moving this check to `.travis.yml` so that it is done once instead of in each file run in CI. Something like (not sure about exit code):
```yaml
script:
  - set -e
  - >
      [[ "$CHECK_ABI_API_COMPLIANCE" == "ON" ]]
      && docker exec -w /home/tango/src cpp_tango .travis/check-ABI-API-compliance.sh
      || true
 - proceed as usual
```

Reply:
I'll be implenting that via github actions, so this will go away.